src/main/scala/filter2.scala

```diff
@@ -1,19 +1,18 @@
 package era7bio.db.rna16s
 
 import era7bio.db._, csvUtils._, collectionUtils._
-
 import ohnosequences.fastarious.fasta._
 import ohnosequences.statika._
-
 import com.github.tototoshi.csv._
 import better.files._
 
-case object filter2 extends FilterDataFrom(filter1)() {
+// TODO I don't understand what this is trying to do, and how.
```

Comment:
It's a bit confusing. First I asked if you want the general description and you said 

> not really. That's about what this needs to do, but not how

Now it's another way round..